.github/workflows/dockerhub-publish-nightly.yml

```
@@ -19,12 +19,14 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Set datetime
+      - name: Set environment variables
```

Shall stay the same style with the style of other step name?
```suggestion
      - name: Set Environment Variables
```